build.gradle

```diff
@@ -113,7 +113,7 @@ dependencies {
     compile 'org.liquibase:liquibase-core:3.6.3'         // For upgrade
     compile 'org.postgresql:postgresql:42.2.5'           // Postgres jdbc driver
     compile 'org.slf4j:slf4j-api:1.7.25'                 // Logging facade
-    compile "org.springframework.boot:spring-boot-starter-web:${springBootVersion}"
+    compile "org.springframework.boot:spring-boot-starter-web"
```

Comment:
Why unlock the spring boot version and other versions?

Reply:
It causes an AutoConfiguration error because of version mismatches. The recommendation is to specify it how I did above.